dask/bag/core.py

```diff
@@ -1928,18 +1928,19 @@ def bag_zip(*bags):
 
     Incorrect usage:
 
-    >>> numbers = db.range(20) # doctest: +SKIP
-    >>> fizz = numbers.filter(lambda n: n % 3 == 0) # doctest: +SKIP
-    >>> buzz = numbers.filter(lambda n: n % 5 == 0) # doctest: +SKIP
-    >>> fizzbuzz = db.zip(fizz, buzz) # doctest: +SKIP
-    >>> list(fizzbuzzz) # doctest: +SKIP
+    >>> numbers = db.range(31, npartitions=1)
+    >>> fizz = numbers.filter(lambda n: n % 3 == 0)
+    >>> buzz = numbers.filter(lambda n: n % 5 == 0)
+    >>> fizzbuzz = db.zip(fizz, buzz)
+    >>> list(fizzbuzz)
     [(0, 0), (3, 5), (6, 10), (9, 15), (12, 20), (15, 25), (18, 30)]
 
     When what you really wanted was more along the lines of the following:
 
-    >>> list(fizzbuzzz) # doctest: +SKIP
-    [(0, 0), (3, None), (None, 5), (6, None), (None 10), (9, None),
-    (12, None), (15, 15), (18, None), (None, 20), (None, 25), (None, 30)]
+    >>> list(fizzbuzz) # doctest: +SKIP
+    (0, 0), (3, None), (None, 5), (6, None), (9, None), (None, 10),
+    (12, None), (15, 15), (18, None), (None, 20),
+    (21, None), (24, None), (None, 25), (27, None), (30, 30)
```

Comment:
Re the following comment, it's not clear to me what the original author was going for, so I took my best guess.
> When what you really wanted was more along the lines of the following

Reply:
huh yeah. I am not sure either. I guess what you have is fine but just wrap it in square brackets like the original.

Reply:
Will do. 

Only problem is I'd accidentally deleted my fork so can't push changes to the feature branch. As seen below, the feature branch is grayed out and can no longer be clicked on (normally clicking on it takes you to the fork). This applies to #7864 as well.
![image](https://user-images.githubusercontent.com/17503044/127057376-413ba1e4-a632-4d78-acf5-114c410b0421.png)
 
Any idea what I can do here without having to open a _new_ PR? I do still have access to the fork locally so no work is lost but I'd prefer not to open a new PR if possible.

Reply:
Oh wierd. I am going to see if I can still push.

Reply:
Nope.

Reply:
I'm just going to merge this since the last comment is about a skipped doctest output.